src/utils.js

```diff
@@ -39,15 +39,25 @@ export function readableNumber(num) {
   } else if (num >= 10000) {
     return `${(num / 1000).toFixed(1)}k`;
   } else if (num < 10000) {
-    return digitGrouping(num);// using only digitGrouping because we only need to display any number below 10k with only a comma
+    return digitGrouping(num); // using only digitGrouping because we only need to display any number below 10k with only a comma
   }
   return num;
 }
 /**
- * 
+ * takes any number over 1k with a dot and turns it into a comma  e.g.(without: 1.000, with: 1,000)
```

Comment:
That's not what it does, it just groups digits. 

instead of `123123123`, it does `123,123,123`.

Reply:
Oh, okay.
On it